usmqe_tests/web/others/test_import_cluster.py

```diff
@@ -6,73 +6,33 @@
 
```

Comment:
Based on further explanation, I think we should remove `others/test_import_cluster.py `, because I think that there should be no gluster or ceph related test cases in others.